generators/_lib/util.js

```diff
@@ -0,0 +1,104 @@
+const forge = require('node-forge');
+const axios = require('axios');
+
+const JAVA_BDK_VERSION_DEFAULT = '2.14.1'
+const PYTHON_BDK_VERSION_DEFAULT = '2.6.1'
+const WDK_VERSION_DEFAULT = '1.6.3'
+const SPRING_VERSION_DEFAULT = '2.7.12'
```

Comment:
since we migrate to jdk17, should we move to spring 3 ? or you think to do so in a next step ?

Reply:
the version check is excluding `RC` and `alpha` tags for now. once we graduate the BDK RC, then spring should default to 3.

https://github.com/finos/generator-symphony/blob/feature/ext-app-overhaul/generators/_lib/util.js#L41